modin/pandas/io.py

```diff
@@ -610,16 +610,48 @@ def read_spss(
     )
 
 
-@_inherit_docstrings(pandas.to_pickle, apilink="pandas.to_pickle")
+# adding docstring since pandas docs doesn't have web section for this function
```

Comment:
Why do we can't to do by this way?

```suggestion
@_inherit_docstrings(pandas.Series.to_pickle, apilink="pandas.Series.to_pickle")
```

Reply:
Discussed [here](https://github.com/modin-project/modin/pull/4375#discussion_r848336160).